rbd/rbd.go

```diff
@@ -1065,8 +1065,8 @@ func (image *Image) SetMetadata(key string, value string) error {
 	return nil
 }
 
-// int rbd_metadata_remove(rbd_image_t image, const char *key)
-func (image *Image) RemoveMetadata(key string) error {
+// RemoveMetadataKey will remove specified key from the image metadata.
+func (image *Image) RemoveMetadataKey(key string) error {
```

Comment:
I think the name is based on the API provided by librbd, namely `rbd_metadata_remove()`. 

Renaming an existing public function can break applications using go-ceph. It is probably not a good idea to rename this.

Reply:
@nixpanic you are right. Seems like I was a little tired last evening. Thanks for the review I will close this pr.